build/separate.md

```diff
@@ -81,3 +81,11 @@ install them into `$HOME/install`, then a configure command line for this
 looks something like this:
 
     LDFLAGS="-Wl,-rpath,$HOME/install/lib" ./configure ...
+
+If your system supports the runpath form of rpath it is often better to use
+that instead because it can be overridden by the LD_LIBRARY_PATH environment
```

Comment:
I think you want to backtick this: ` LD_LIBRARY_PATH`. But rpath and runpath *probably* needs to be added as acceptable words for the spellcheck.